### DISCUSSION TITLE
Ignore some request for the count of Inactivity Timeout

### DISCUSSION THREAD

**Discussion by gixlg** [OpenIDC/mod_auth_openidc#517] [Q&A] [answered]
Hi,
I have this use case: I want to invalidate the session after that the user does not perform any action after 30 min.   
To achieve this goal I found, in the documentation, this configuration: ```OIDCSessionInactivityTimeout``` that is exactly what I was looking for.

But, the problem here is that inside my application there is a javascipt code that do a Ajax request to certain URL, let's say ```/api/notification```, to get the list of new notification that the user has.
The request is in polling mode, fired every 5 min. This call keeps the session fresh even if the users aren't using the application.

There is a way to ignore this request for the count of Inactivity Timeout?
There are any filtering options related to that configuration?
What approach do you suggest for this situation?

Thanks :)

**Comment by zandbelt** [comment, marked as answer]
No, there's no way to do that.

**Reply by OlivierBOEL** [reply]
Alternatively, I think you can just let /api/notification not protected
<Location /api/notification >
        Require all granted
</Location>
Not the best option from a security perspective.
